app/controllers/user/listings_controller.rb

```diff
@@ -20,8 +20,8 @@ def create
       params["listing_images"]["images"].each do |i|
         @listing.listing_images.build(image: i)
       end
-    end
-    if @listing.save
+    elsif @listing.save
+      current_user.update_attributes(role: 1)
```

Comment:
This is where we change a users role to a host role when a listing is created, this was included in my last pul request but someone deleted it. Without it roles dont change.